Gemfile.lock

```diff
@@ -3,6 +3,7 @@ PATH
   specs:
     retest (1.13.0)
       listen (~> 3.9)
+      observer (~> 0.1.0)
```

Comment:
@AaronRustad 
Sorry, I wasn't clear enough; the minor version should be updated here, too. 
Bundling again locally will probably update the Gemfile.lock accordingly.

Once this is done I will merge and release your changes as `v1.13.1`

Thanks @AaronRustad

Reply:
Gah. Of course!